shared/oae/css/oae.base.css

```diff
@@ -197,6 +197,40 @@ a:hover, .btn-link:hover, a:focus, .btn-link:focus {
     margin-top: 1px;
 }
 
+/* HR-like element that has a split with text in the middle of it */
+
+div.hr {
```

Comment:
Did we get this from a stackoverflow question? If so, it would be worth adding the URL in an `@see` statement